### PR TITLE
Adding Requirements

### PR DESCRIPTION
**Pull request by kuldeepsidhu88**

### CODE REVIEW

**Comment by kuldeepsidhu88** [comment]
@santoshbk Review these and update/refine these if you think of anything else.

File: Requirements.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# book-club
+Projects idea is to create an repo for books, which registered users can like, rate and review.
```
Comment (santoshbk) [inline Requirements.md]: The idea of project is to create a repo for books, which registered users can like, rate and review.

Comment (santoshbk) [reply Requirements.md]: Just a grammar mistake, to make sure i was reading everything properly!

**Review by santoshbk — CHANGES_REQUESTED** [review]
Please correct the changes suggested, so that i can approve and merge! 
thanks

**Comment by kuldeepsidhu88** [comment]
@santoshbk Merging this PR as this looks in good state now.

**Pull request merged**